dev/com.ibm.ws.concurrent.mp.1.0/src/com/ibm/ws/concurrent/mp/ManagedCompletableFuture.java

```diff
@@ -1333,7 +1333,7 @@ final boolean super_cancel(boolean mayInterruptIfRunning) {
      *
      * @see java.util.concurrent.CompletableFuture#complete(T)
      */
-    final boolean super_complete(T value) {
+    public final boolean super_complete(T value) {
```

Comment:
This and the other super_completExceptionally method is what I would really like to avoid making public, because if it gets invoked without our knowledge, it could corrupt state.  With the current split, I don't see a way to avoid it either.

Reply:
Resolved with the latest version of the changes.

Reply:
Resolved with the latest version of the changes.